beacon_node/beacon_chain/src/light_client_server_cache.rs

```diff
@@ -195,6 +209,50 @@ impl<T: BeaconChainTypes> LightClientServerCache<T> {
         Ok(())
     }
 
+    fn store_current_sync_committee_branch(
+        &self,
+        store: &BeaconStore<T>,
+        cached_parts: &LightClientCachedData<T::EthSpec>,
+        block_root: Hash256,
+    ) -> Result<(), BeaconChainError> {
+        let column = DBColumn::SyncCommitteeBranch;
+        store.hot_db.put_bytes(
+            column.into(),
+            &block_root.as_ssz_bytes(),
+            &cached_parts.current_sync_committee_branch.as_ssz_bytes(),
+        )?;
+        Ok(())
+    }
+
+    fn store_sync_committee(
+        &self,
+        store: &BeaconStore<T>,
+        cached_parts: &LightClientCachedData<T::EthSpec>,
+        sync_committee_period: u64,
+        finalized_period: u64,
+    ) -> Result<(), BeaconChainError> {
+        let column = DBColumn::SyncCommittee;
+
+        // TODO should i always write to db, even if a record already exists?
```

Comment:
This implementation is not handling re-orgs. So there can exist a record that became orphaned. You can keep a hash of the latest persisted sync committee and only persist if the hash changes

Reply:
im now only persisting if the current_sync_committee has changed